libvmi/driver/xen/xen_private.h

```diff
@@ -55,8 +55,10 @@
 #define xen_pfn_to_cr3_x86_32(pfn) (((unsigned)(pfn) << 12) | ((unsigned)(pfn) >> 20))
 #define xen_cr3_to_pfn_x86_32(cr3) (((unsigned)(cr3) >> 12) | ((unsigned)(cr3) << 20))
 
+#ifndef HAVE_XC_DOMAIN_MAXIMUM_GPFN
```

Comment:
Since this is now it's own configure check, it might be less confusing to pull this outside of the `#ifndef xen_cr3_to_pfn_x86_32`